components/card/w-card.vue

```diff
@@ -1,29 +1,27 @@
 <script setup>
-import { card as ccCard } from '@warp-ds/css/component-classes';
 import { computed } from 'vue';
 
+import { card as ccCard } from '@warp-ds/css/component-classes';
+
 defineOptions({
   name: 'wCard',
 });
 
 const props = defineProps({
   as: { type: String, default: 'div' },
   selected: Boolean,
-  flat: Boolean
-})
+  flat: Boolean,
+});
 
 const containerClasses = computed(() => ({
   [ccCard.card]: true,
   [ccCard.cardShadow]: !props.flat,
   [ccCard.cardSelected]: !props.flat && props.selected,
   [ccCard.cardFlat]: props.flat,
-  [props.selected ? ccCard.cardFlatSelected : ccCard.cardFlatUnselected]: props.flat
+  [props.selected ? ccCard.cardFlatSelected : ccCard.cardFlatUnselected]: props.flat,
 }));
 
-const outlineClasses = computed(() => [
-  ccCard.cardOutline,
-  props.selected ? ccCard.cardOutlineSelected : ccCard.cardOutlineUnselected,
-]);
+const outlineClasses = computed(() => [ccCard.cardOutline, props.selected ? ccCard.cardOutlineSelected : ccCard.cardOutlineUnselected]);
```

Comment:
Optimally it should leave these kind of things untouched... 🫤 Makes readability a lot worse to have it on one line.

Reply:
I'll also see if I can add a rule in the .eslintrc-file. I think it is the `vue-eslint-parser` that needs to be tweaked.

Reply:
Same issue as this: https://github.com/warp-ds/vue/pull/167#discussion_r1609863709